bin/coreos-install

```diff
@@ -380,11 +360,35 @@ do
         f) IMAGE_FILE="$OPTARG" ;;
         n) COPY_NET=1;;
         v) set -x ;;
-        h) echo "$USAGE"; exit;;
+        h) usage=1;;
         *) exit 1;;
     esac
 done
 
+if [[ -n ${usage} ]]; then
+	echo "
```

Comment:
Nitpicking: Maybe start this with `echo "\` so that it doesn't begin printing an extra newline.  This also looks like it starts mixing in tab indentation where the rest of the file is spaces.  (We're not particularly consistent about that, I know.)